### PR TITLE
Switch to main server logger, always set a logger

### PR DESCRIPTION
**Pull request by sethvargo**
Switch to main server logger, always set a logger

**Release Note**

```release-note
Use shared logging package, always configure a logger
```

### CODE REVIEW

File: pkg/controller/middleware/method.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ import (
 	"net/http"
 	"strings"
 
-	"github.com/google/exposure-notifications-verification-server/pkg/logging"
+	"github.com/google/exposure-notifications-server/pkg/logging"
```
Comment (icco) [inline pkg/controller/middleware/method.go:22]: nit: consistent whitespace missing.

File: cmd/add-users/main.go [hunk 2]
```diff
@@ -18,67 +18,89 @@ package main
...
+		logger.Fatal(err)
+	}
+}
+
+func realMain(ctx context.Context) error {
+	logger := logging.FromContext(ctx)
+
 	if len(flag.Args()) > 0 {
-		log.Fatal("Received unexpected arguments:", flag.Args())
+		return fmt.Errorf("unexpected arguments: %v", flag.Args())
```
Comment (icco) [inline cmd/add-users/main.go:60]: nit: %w

File: pkg/controller/middleware/method.go (hunk 1 above)
Comment (sethvargo) [reply pkg/controller/middleware/method.go:22]: It's actually consistent now 😄 . It goes:

- Group 0: stdlib package
- Group 1: internal packages
- Group 2: main server packages
- Group 3: external packages

We were pretty inconsistent about it, so I fixed it in all the files that this touches.

Comment (icco) [reply pkg/controller/middleware/method.go:22]: yeah, there's no whitespace between group 2 and 3 in this file, while there is in all other files in this commit.

**Pull request merged**